functions.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+
+/**
+ * connects SQL database to PHP
+ */
+function connectDB()
+{
+    $db = new PDO ('mysql:host=db; dbname=AlexCollection', 'root', 'password');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    return $db;
+}
+
+/**
+ * pulls set names from PDO
```

Comment:
Try to make the description more generic, technically could pass any db connection and should work. Make the description about what the function does eg: Requests card set details from a database